purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -125,6 +126,15 @@ class Purchases internal constructor(
 
     // region Public Methods
 
+    /**
+     * Allows the SDK to handle the activities intent to process relevant deep links.
+     * @return A boolean indicating whether the SDK handled the deep link or ignored it.
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    fun handleDeepLink(intent: Intent): Boolean {
```

Comment:
Note that this API means that the SDK needs to be configured BEFORE handling the deep link. I think we would need to add a wrapper around this to handle cases where the SDK is not configured.